components/Tabs.tsx

```diff
@@ -113,8 +99,8 @@ const Tabs = (props) => {
       </ul>
 
       <div className='tab-content'>
-        {children.map((child) => {
-          if (child.props.label !== activeTab) return undefined
+        {children!.map((child) => {
+          if (child.props.htmlFor !== activeTab) return undefined
```

Comment:
return `null`. returning `undefined` is not the preferred way.

Reply:
Thanks, I will do that